example/src/app.js

```diff
@@ -21,7 +21,5 @@ export const App = (props, { state, effects }) => {
   );
 };
 
-App.contextTypes = contextTypes;
 
-
-export default addState(App);
+export default addState(injectState(App));
```

Comment:
It isn't clear from the example (I can fix that) but `withState` and `injectState` serve two different functions.  `withState` sets up the state container, with `effects`, `state`, etc.  `injectState` subscribes to state changes and injects `state` and `effects` as props into its wrapped component.  I'm considering adding a flag as an option to `withState` in case you _really do_ want to update the whole subtree whenever the state changes (the old behavior).  But composing in the above fashion doesn't seem too bad, and at least it is consistent.